.github/workflows/tav-command.yml

```diff
@@ -18,35 +18,66 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
     outputs:
-      versions: ${{ steps.generate-matrix.outputs.versions }}
-      modules: ${{ steps.generate-matrix.outputs.modules }}
-      pull-request: ${{ steps.transform.outputs.pull-request }}
+      versions: ${{ steps.transform.outputs.versions }}
+      modules: ${{ steps.transform.outputs.modules }}
+      pull-request: ${{ steps.pr.outputs.number }}
     steps:
-    - name: Validate github comment
-      uses: elastic/apm-pipeline-library/.github/actions/validate-github-comment@current
+      - name: Validate github comment
+        uses: elastic/apm-pipeline-library/.github/actions/validate-github-comment@main
 
-    - id: transform
-      name: Transform comment to the supported matrix
-      uses: actions/github-script@v6
-      with:
-        script: |
-          core.setOutput('pull-request', context.payload.issue.number)
+      - id: pr
+        name: Fetch PR
+        uses: actions/github-script@v6
+        with:
+          script: core.setOutput('number', context.payload.issue.number)
 
-    - uses: actions/checkout@v3
-      with:
-        ref: refs/pull/${{ steps.transform.outputs.pull-request }}/merge
+      - uses: actions/checkout@v3
+        with:
+          ref: refs/pull/${{ steps.pr.outputs.number }}/merge
 
-    ## TODO: use elastic/apm-pipeline-library/.github/actions/version-framework
-    ##       as soon as https://github.com/elastic/apm-pipeline-library/issues/2171 is done.
-    - id: generate-matrix
-      run: .ci/generate-github-tav-matrix.sh
+      - id: transform
+        name: Transform comment to the supported matrix
+        uses: actions/github-script@v6
+        with:
+          script: |
+            async function f() {
+              const fsPromises = require('fs').promises;
+              const data = await fsPromises.readFile('./.ci/tav.json')
+                                 .catch((err) => core.setFailed('Failed to read file', err));
+              return JSON.parse(data.toString());
+            }
+            const matrix = await f()
+            let modules, versions
+            try {
+              versions = matrix.versions
+              modules = matrix.modules
+            } catch (err) {
+              core.setFailed("Error parsing JSON string")
+              return
+            }
+            const comment = context.payload.comment.body
+            if (comment !== '/test tav') {
+              const regex = /\/test tav ([^\s\\]+)(\s*)([^\s\\]*)/
+              const match = comment.match(regex)
+              console.log(`parse comment - ${match}`)
+              if (match[1]) {
+                if (match[1] !== 'all') {
+                  modules = match[1].split(',')
+                }
+              }
+              if (match[3]) {
+                versions = match[3].split(',')
+              }
```

Comment:
This should guard against the `match` possibly being `null`, otherwise `match[1]` could throw an error.

```js
> const regex = /\/test tav ([^\s\\]+)(\s*)([^\s\\]*)/

> comment = '/test tavern'
'/test tavern'
> match = comment.match(regex)
null
> match[0]
Uncaught TypeError: Cannot read properties of null (reading '0')

> comment = '/test tav\nblah blah blah\n'
'/test tav\nblah blah blah\n'
> match = comment.match(regex)
null
```

Reply:
https://github.com/elastic/apm-agent-nodejs/pull/3246/commits/9e3c64a7bd1458a927caf6cf6cf684373cb3ba2d is the guard and 
https://github.com/elastic-robots/gh-actions-interactions/actions/runs/4626017105 was caused with https://github.com/elastic-robots/gh-actions-interactions/pull/3#issuecomment-1498544288